personality/se_dpm/service_puppet.pan

```diff
@@ -4,46 +4,59 @@ variable SEDPM_CONFIG_SITE ?= error("SEDPM_CONFIG_SITE should be defined");
 
 include SEDPM_CONFIG_SITE;
 
-variable SEDPM_MONITORING_ENABLED?=false;
-variable XROOT_ENABLED=true;
-variable HTTPS_ENABLED?=false;
-variable DPM_MEMCACHED_ENABLED?=false;
+variable SEDPM_MONITORING_ENABLED ?= false;
+variable XROOT_ENABLED ?= true;
+variable HTTPS_ENABLED ?= false;
+variable DPM_MEMCACHED_ENABLED ?= false;
 
-variable VOMS_XROOTD_EXTENSION_ENABLED = true;
+variable VOMS_XROOTD_EXTENSION_ENABLED ?= true;
 
-variable XROOTD_FEDERATION_SITE_CONFIG?=null;
+variable XROOTD_FEDERATION_SITE_CONFIG ?= null;
 
 variable SEDPM_IS_HEAD_NODE ?= if( FULL_HOSTNAME == DPM_HOSTS['dpm'][0] ) {
-                                 true;
-                               } else {
-                                 false;
-                               };
+    true;
+}else{
+    false;
+};
```

Comment:
The `true` and `false` are pointless here, this can be simplified down to:
```pan
variable SEDPM_IS_HEAD_NODE ?= FULL_HOSTNAME == DPM_HOSTS['dpm'][0];
```